python/test/test_dataset.py

```diff
@@ -267,14 +267,40 @@ def test_binary_operations(self):
         self.assertTrue(np.array_equal(c[Data.Value, "i"].numpy, data * 2.0))
         c = a / 2.0
         self.assertTrue(np.array_equal(c[Data.Value, "i"].numpy, data / 2.0))
+        c = 2.0 + a
+        self.assertTrue(np.array_equal(c[Data.Value, "i"].numpy, data + 2.0))
+        c = 2.0 - a
+        self.assertTrue(np.array_equal(c[Data.Value, "i"].numpy, 2.0 - data))
+        c = 2.0 * a
+        self.assertTrue(np.array_equal(c[Data.Value, "i"].numpy, data * 2.0))
+
+        d = Dataset()
+        d[Coord.X] = ([Dim.X], np.arange(10))
+        d[Data.Value, "i"] = ([Dim.X], np.arange(10, dtype='float64'))
+        a_slice = a[Dim.X, :]
+        d_slice = d[Dim.X, :]
+
+        # Equal
```

Comment:
Can you test these in a different test method?

Reply:
Do you mean in a new `def test_equal_and_not_equal(self)` function inside the same file?

Reply:
Yes. I think that would help readability and debugging future test failures.